specs/MultiProfile.md

```diff
@@ -296,6 +306,8 @@ namespace Microsoft.Web.WebView2.Core
         Boolean IsInPrivateModeEnabled { get; };
 
         String ProfilePath { get; };
+
+        CoreWebView2CookieManager cookieManager { get; };
```

Comment:
After making the change to the COM interface to have CookieManager in a new COM interface, you should see a similar change in this generated MIDL3 with CookieManager in an interface_name block within the CoreWebView2Profile runtimeclass:
```
        [interface_name("Microsoft.Web.WebView2.Core.ICoreWebView2Profile2")]
        {
            CoreWebView2CookieManager CookieManager { get; };
        }
```

Reply:
Hi, I am not familiar with MIDL3 generation. Could you please tell me how to get it?